### ISSUE TITLE
Documentation: Message types not reversed in Function <-> Eff error section

### ISSUE THREAD

**Issue by liammcdermott** [purescript/purescript#2287]
There seems to be a minor copy/paste error on this page: https://github.com/purescript/purescript/wiki/Error-Code-TypesDoNotUnify

The two error messages should have their respective types reversed, but the error messages in both samples are the same.

Proposed fix: change the last error message sample to read:

`Could not match type

  Function

with type

  Eff`

I could fix it myself, but I've already erroneously reported a couple of things as bugs to @paf31 that weren't, so am nervous about editing documentation without guidance.

**Comment by jplatte** [comment]
Huh, I listed this in my issue (#2187) as well, someone ticked the checkbox as completed, but I'm pretty sure it was the exact same when I last looked at that document.

**Comment by paf31** [comment]
Ah that was me. I thought I fixed it, but now it looks like I didn't. I'll close this, but please do update the wiki as you see fit.

**Comment by jplatte** [comment]
What? You still didn't address it. I now checked your last two revisions, they were on completely different things. But this time I just reproduced one of the error messages and edited the page accordingly (before I wasn't sure which code block needed to have the types reversed). [Here's the diff with the section highlighted](https://github.com/purescript/purescript/wiki/Error-Code-TypesDoNotUnify/_compare/9b8a65b9f0109e57714738fde48af85582f9e929...a0a2a8b05c45aeb5bcba05c2f1f58fab9766b0fc#diff-a27bbcbbce02b395f12a7e40d7aa353cL52) to finally clear this up.
